### PR TITLE
fix: white block when keyboard is open

### LINKED ISSUE #206: After hidden bottom tab bar by open keyboard remains white block
### Current behaviour
After hidden bottom tab bar by open keyboard remains white block. I use mode `android:windowSoftInputMode="adjustResize"` and `adjustResize` fits most to me. I tried other modes and not fits me.

### Expected behaviour

I expect to hide the white bar in adjustResize mode

### Code sample

```
import React from 'react';
import { Input } from "react-native-elements";
import { View, ScrollView } from 'react-native';

class Example extends React.Component {
    render() {
        return (
            <View style={ { backgroundColor: '#ccc', flex: 1 } }>
                <ScrollView>
                    <Input
                        label={ 'Серия бланка' }
                        placeholder='Серия бланка'
                    />
                </ScrollView>
            </View>
        );
    }
}
```

### Screenshots (if applicable)

photo_2019-10-24_20-37-34

### What have you tried
I watched #16 / #64 

### Your Environment

| software              | version
| --------------------- | -------
| ios or android        | android
| react-native          | 0.60.5
| react-navigation-tabs | 4.0.10
| react-navigation-stack | 1.10.2
| react-navigation-tabs | 2.5.6
| node                  | v10.17.0
| npm or yarn           | 6.11.3

### PR DESCRIPTION
**Pull request by vonovak**
### Motivation

this fixes #206

### Test plan

tested locally, please see the videos here [before-and-after.zip](https://github.com/react-navigation/tabs/files/3876932/before-and-after.zip)

related https://github.com/react-navigation/tabs/commit/c8e1cd24bd92bc51ed916d2e07feae39be3b3757#diff-df8db9f0a4883d10ede795953bd329bb

### CODE REVIEW

**Comment by fri0z** [comment]
@vonovak Indeed, this fix solves the problem. I tested on my environment.

**Comment by fri0z** [comment]
@satya164 or @ahmedu007 pay attention to this request.

File: src/views/BottomTabBar.tsx [hunk 1]
```diff
@@ -457,12 +457,14 @@ class TabBarBottom extends React.Component<BottomTabBarProps, State> {
...
-                // Absolutely position the tab bar so that the content is below it
-                // This is needed to avoid gap at bottom when the tab bar is hidden
-                position: this.state.keyboard ? 'absolute' : null,
               }
             : null,
           containerStyle,
+          {
+            // Absolutely position the tab bar so that the content is below it
+            // This is needed to avoid gap at bottom when the tab bar is hidden
+            position: this.state.keyboard ? 'absolute' : undefined,
```
Comment (satya164) [inline src/views/BottomTabBar.tsx]: Thanks for the PR.

This isn't the correct place to put it. Coz we are updating `this.state.keyboard` regardless `keyboardHidesTabBar` enabled or not and we won't want to set position to `absolute` when this setting is not enabled.

I think the bug is in the ordering of styles. Moving `containerStyle` before the style related to `keyboardHidesTabBar` should fix this. Can you update the PR with the change?

Comment (vonovak) [reply src/views/BottomTabBar.tsx]: @satya164 that's right, thanks. I reordered those styles and tested with `keyboardHidesTabBar` true / false; it looks good.

**Pull request merged**